fe/fe-core/src/main/java/com/starrocks/catalog/MaterializedView.java

```diff
@@ -61,7 +61,8 @@ public class MaterializedView extends OlapTable implements GsonPostProcessable {
     public enum RefreshType {
         SYNC,
         ASYNC,
-        MANUAL
+        MANUAL,
+        REALTIME
```

Comment:
which is better: `INCREMENTAL`  or `REALTIME `? I doubt we now just implement the incremental computations rather than realtime?

Reply:
or `AUTO_INCREMENTAL`, and then we can introduce another way, `AUTO_REALTIME`

Reply:
Changed to `INCREMENTAL`.